lib/phoenix/verified_routes.ex

```diff
@@ -108,32 +110,77 @@ defmodule Phoenix.VerifiedRoutes do
       such as library code, or application code that relies on multiple routers. In such cases,
       the router module can be provided explicitly to `path/3` and `url/3`.
 
-  ## Tracking Warnings
+  ## Tracking warnings
 
   All static path segments must start with forward slash, and you must have a static segment
   between dynamic interpolations in order for a route to be verified without warnings.
-  For example, the following path generates proper warnings
+  For example, imagine you have these two routes:
 
-      ~p"/media/posts/#{post}"
+      get "/media/posts/:id"
+      get "/media/images/:id"
 
-  While this one will not allow the compiler to see the full path:
+  The following route will be verified and emit a warning as it does not match the router:
 
-      type = "posts"
+      ~p"/media/post/#{post}"
+
+  However the one below will not, the "post" segment is dynamic:
+
+      type = "post"
       ~p"/media/#{type}/#{post}"
 
-  In such cases, it's better to write a function such as `media_path/1` which branches
-  on different `~p`'s to handle each type.
+  If you find yourself needing to generate dynamic URLs which are defined statically
+  in the router, that's a good indicator you should refactor it into one or more
+  function, such as `posts_path/1` and `images_path/1`.
 
   Like any other compilation warning, the Elixir compiler will warn any time the file
-  that a ~p resides in changes, or if the router is changed. To view previously issued
-  warnings for files that lack new changes, the `--all-warnings` flag may be passed to
-  the `mix compile` task. For the following will show all warnings the compiler
-  has previously encountered when compiling the current application code:
+  that a `~p` resides in changes, or if the router is changed.
+
+  ## Localized routes and path prefixes
+
+  Applications that need to support internationalization (i18n) and localization (l10n)
+  often do so at the URL level. In such cases, there are different approaches one can
+  choose.
+
+  One option is to perform i18n at the domain level. You can have `example.com` (in which
+  you would detect the locale based on the "Accept-Language" HTTP header), `en.example.com`,
+  `en-GB.example.com` and so forth. In this case, you would have a plug that looks at the
+  host and at HTTP headers and calls `Gettext.get_locale/1` accordingly. The biggest benefit
+  of this approach is that you don't have to change the routes in your application and
+  verified routes works as is.
+
+  Some applications, however, like to the locale as part of the URL prefix:
```

Comment:
Missing **add** in the sentence.

"Some applications, however, like to **add** the locale as part of the URL prefix:"